tests/integration/pkg/resource/manager.go

```diff
@@ -150,6 +233,24 @@ func (m *Manager) UpdateResource(client dynamic.Interface, resourceSchema schema
 	}, m.retryOptions...)
 }
 
+// UpdateResourceWithoutNS updates a given k8s resource without namespace
+func (m *Manager) UpdateResourceWithoutNS(client dynamic.Interface, resourceSchema schema.GroupVersionResource, name string, updateTo unstructured.Unstructured) error {
+	return retry.Do(func() error {
+		time.Sleep(5 * time.Second) //TODO: delete after waiting for resource creation is implemented
```

Comment:
There is still a TODO. What do we do with this?

Reply:
It's also in other functions so I guess waiting for resources is not implemented yet